### ISSUE TITLE
Canvas Support

### ISSUE THREAD

**Issue by caglarongan** [mdelobelle/obsidian_supercharged_links#166]
Being able to change embeded cards' colors based on tags, metadata, filepath just like links.

Alternative / Extra:
Being able to *supercharge* cards' inline title (when active) or filename on top.

***
This plugin helps a ton in terms of *skimming* easily all around the app. Since Canvas is about birds' eye view and laying down notes next to each other, Supercharged Links would be a great for that.
For example, distinguishing literature notes, permanent notes and fleeting notes when zoomed out.

**Comment by HEmile** [comment]
I think this needs to be a bit more specific. Automatically changing the card colour could be confusing when users attempt to manually set it. And I'm not sure what you mean with the cards title

**Comment by caglarongan** [comment]
Good point. Maybe not changing the whole color but only changing  border's color. I'm not sure, I can't think of a better way at the moment. 

This is what I meant by cards' title:
![SCR-20230109-j91](https://user-images.githubusercontent.com/112904909/211344287-c49e48bd-c062-4a23-ae21-0183189ca46c.png)
